libs/ui-lib/lib/ocm/components/clusterConfiguration/operators/CnvHostRequirements.tsx

```diff
@@ -23,7 +29,14 @@ const CnvHostRequirements = ({ clusterId }: { clusterId: Cluster['id'] }) => {
 
   const workerRequirements = cnvRequirements?.requirements?.worker?.quantitative;
   const masterRequirements = cnvRequirements?.requirements?.master?.quantitative;
-
+  const odfText = !isSingleNode
+    ? 'OpenShift Data Foundation (recommended for creating additional on-premise clusters)'
+    : '';
+  const lvmsText = showMessageForLvms
+    ? !isSingleNode
+      ? ', Logical Volume Manager Storage'
+      : 'Logical Volume Manager Storage'
+    : '';
```

Comment:
I'm sorry, I'm really lost in all of these ternary operators. I think for the sake of readability, it would be better to do something like:

```js
let operatorInfo: string;

if (condition1) {
    operatorInfo = "Message when condition1";
} else if (condition2) {
    operatorInfo = "Message when condition2";
}
```

```js

<ListItem>
{operatorInfo}
</ListItem>
```

I know there will be some duplication with the strings but it should be much clearer than piecing it together on three different places.